.env.production

```
@@ -0,0 +1,222 @@
+
+# Cloud storage (recommended for production)
+AWS_S3_BUCKET=skyyrose-production
+AWS_ACCESS_KEY_ID=REPLACE_WITH_AWS_KEY
+AWS_SECRET_ACCESS_KEY=REPLACE_WITH_AWS_SECRET
+AWS_REGION=us-west-2
+
+# ============================================================================
+# BACKUP CONFIGURATION
+# ============================================================================
+BACKUP_ENABLED=true
+BACKUP_SCHEDULE=0 1 * * *
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)